.travis.yml

```diff
@@ -0,0 +1,41 @@
+language: java
+jdk: openjdk8
+sudo: required
+dist: trusty
+
+install:
+- wget http://www-us.apache.org/dist/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz
+- tar xzf apache-maven-3.3.9-bin.tar.gz
+- curl -s -L https://github.com/mozilla/smarthome/releases/latest | egrep -o /mozilla/smarthome/releases/download/.*/m2repository.tar.gz | wget --base=http://github.com/ -i -
+- tar xf m2repository.tar.gz -C $HOME/.m2/repository/
+
+before_script:
+- rm $HOME/.m2/settings.xml
+- export MAVEN_OPTS="-Dmaven.artifact.threads=3 -Xmx2469m -XX:MaxPermSize=512m"
+
```

Comment:
As I said: I wanna (re)try - as those two built on my machine a couple of
times without problems. In worst case I'd have to exclude them again.

On Wed, Jun 8, 2016 at 1:33 PM, Kelly Davis <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/mozilla/smarthome/pull/65#discussion_r66239023>:
>
> > @@ -0,0 +1,41 @@
> > +language: java
> > +jdk: openjdk8
> > +sudo: required
> > +dist: trusty
> > +
> > +install:
> > +- wget http://www-us.apache.org/dist/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz
> > +- tar xzf apache-maven-3.3.9-bin.tar.gz
> > +- curl -s -L https://github.com/mozilla/smarthome/releases/latest | egrep -o /mozilla/smarthome/releases/download/.*/m2repository.tar.gz | wget --base=http://github.com/ -i -
> > +- tar xf m2repository.tar.gz -C $HOME/.m2/repository/
> > +
> > +before_script:
> > +- rm $HOME/.m2/settings.xml
> > +- export MAVEN_OPTS="-Dmaven.artifact.threads=3 -Xmx2469m -XX:MaxPermSize=512m"
> > +
>
> So are we sure that the bundles
>
>    - bundles/automation/org.eclipse.smarthome.automation.module.timer.test
>    -
>    bundles/automation/org.eclipse.smarthome.automation.module.script.test
>
> don't fail intermittently? Their build exclusion has been removed from the
> PROJECTS env variable.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mozilla/smarthome/pull/65/files/18350ff7b7c77f7e5c2f4c10c1e81a0f6e022b80#r66239023>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFtqsHGVbynHrSZfuNGONSDIe1l_uzLmks5qJqihgaJpZM4IwrNd>
> .
>